Tasks/Gradle/task.json

```diff
@@ -227,125 +228,36 @@
             "name": "sqAnalysisEnabled",
             "aliases": [ "sonarQubeRunAnalysis" ],
             "type": "boolean",
-            "label": "Run SonarQube Analysis",
+            "label": "Run SonarQube or SonarCloud Analysis",
             "required": true,
             "defaultValue": "false",
             "groupName": "CodeAnalysis",
-            "helpMarkDown": "Run a [SonarQube analysis](https://go.microsoft.com/fwlink/?LinkID=708598) after executing the current goals. 'install' or 'package' goals should be executed first."
+            "helpMarkDown": "Run a [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing the current tasks. You have to add the `Prepare Analysis Configuration` task of the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) or [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) extensions in your build definition before this Gradle task."
```

Comment:
@DavidStaheli : Any feedback/suggestions on this help text?

Reply:
Can we please change this as follows to explain how the behavior changed from version 1 of the Gradle task?

```markdown
This option has changed from version 1 of the **Gradle** task to use the [SonarQube](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube) and [SonarCloud](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) marketplace extensions.  Enable this option to run [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing tasks in the **Tasks** field. You must also add a **Prepare Analysis Configuration** task from one of the extensions to the build definition before this Gradle task.
```